worlds/alttp/Client.py

```diff
@@ -581,31 +581,25 @@ async def game_watcher(self, ctx):
 def get_alttp_settings(romfile: str):
     import LttPAdjuster
 
-    last_settings = Utils.get_adjuster_settings(GAME_ALTTP)
-    base_settings = LttPAdjuster.get_argparser().parse_known_args(args=[])[0]
-    allow_list = {"music", "menuspeed", "heartbeep", "heartcolor", "ow_palettes", "quickswap",
-                  "uw_palettes", "sprite", "sword_palettes", "shield_palettes", "hud_palettes",
-                  "reduceflashing", "deathlink", "allowcollect", "oof"}
-
-    for option_name in allow_list:
-        # set new defaults since last_settings were created
-        if not hasattr(last_settings, option_name):
-            setattr(last_settings, option_name, getattr(base_settings, option_name))
-
     adjustedromfile = ''
-    if last_settings:
+    if vars(Utils.get_adjuster_settings_no_defaults(GAME_ALTTP)):
+        last_settings = Utils.get_adjuster_settings(GAME_ALTTP)
+
+        allow_list = {"music", "menuspeed", "heartbeep", "heartcolor", "ow_palettes", "quickswap",
+                    "uw_palettes", "sprite", "sword_palettes", "shield_palettes", "hud_palettes",
+                    "reduceflashing", "deathlink", "allowcollect", "oof"}
         choice = 'no'
-        if not hasattr(last_settings, 'auto_apply') or 'ask' in last_settings.auto_apply:
+        if 'ask' in last_settings.auto_apply:
             printed_options = {name: value for name, value in vars(last_settings).items() if name in allow_list}
-            if hasattr(last_settings, "sprite_pool"):
-                sprite_pool = {}
-                for sprite in last_settings.sprite_pool:
-                    if sprite in sprite_pool:
-                        sprite_pool[sprite] += 1
-                    else:
-                        sprite_pool[sprite] = 1
-                    if sprite_pool:
-                        printed_options["sprite_pool"] = sprite_pool
+            
+            sprite_pool = {}
```

Comment:
Ditto for removing this special case. Just show the pool as it exists in the yaml